requirements-testing.txt

```diff
@@ -30,4 +30,4 @@ nose>=1.3
 flake8>=2.1.0
 psutil==3.0.0
 gcs-oauth2-boto-plugin
-azure
+azure-storage<=0.20.3
```

Comment:
This doesn't seem necessary to fix the current build break, if so, please consider reverting it to previous text.